test/katex-spec.js

```diff
@@ -1755,8 +1755,8 @@ describe("An accent parser", function() {
         expect(parse.type).toEqual("supsub");
     });
 
-    it("should not parse expanding accents", function() {
-        expect("\\widehat{x}").toNotParse();
+    it("should parse expanding accents", function() {
+        expect("\\widehat{x}").toParse();
```

Comment:
Well, `\widehat`, which Jasmine already tests, happens to be both stretchy and shifty. I will write a test for one that is not shifty, say `\overrightarrow`. And I'll edit the output strings to make it more clear which is which.